job_templates/ci_job.xml.em

```diff
@@ -45,7 +45,7 @@
     <userRemoteConfigs>
       <hudson.plugins.git.UserRemoteConfig>
         <url>@(ci_scripts_repository)</url>
-        <credentialsId>1c2004f6-2e00-425d-a421-2e1ba62ca7a7</credentialsId>
+        <credentialsId>@[if not avoid_credentials]1c2004f6-2e00-425d-a421-2e1ba62ca7a7@[end if]</credentialsId>
```

Comment:
Nitpick: you can avoid the bracket syntax here: `@('1c2004f6-2e00-425d-a421-2e1ba62ca7a7' if not avoid_credentials else '')`.